Rakefile

```diff
@@ -20,14 +20,14 @@ namespace :spec do
   desc "Setup the Database for testing"
   task setup: [:connection] do
     ActiveRecord::Base.connection_pool.with_connection do |conn|
-      conn.create_database ENV.fetch("TEST_DATABASE", "pg_enum_test"), owner: ENV.fetch("TEST_USER") { ENV.fetch("USER", "pg_enum") }
+      conn.create_database db_config[:database], owner: db_config[:username]
```

Comment:
I don't see any real value being added here... CI demands that I be able to override some of these defaults using environment variables

Reply:
So this is just using the db_config that was already setup.  Its using the same environment variables, just not checking them twice.

Reply:
Gotcha. Turns out I needed to change this API slightly for 6.1 but I'll accept a patch that uses `db_config_hash`